### DISCUSSION TITLE
struggling to install on Debian bookworm

### DISCUSSION THREAD

**Discussion by mchester50** [TheAssassin/AppImageLauncher#660] [Q&A]
I have tried to install AppimageLauncher using "sudo add-apt-repository ppa:appimagelauncher-team/stable and I receive a message File "/usr/bin/add-apt-repository", line 362, in <module>
sys.exit(0 if addaptropo.main() else 1)" addaptropo.main has uparrows under it. When I install it using the .deb package all works until I try to integrate and run at which point I receive the following "Error-AppImageLauncher failed to register AppImage in system via libappimage. What am I doing wrong? Not this is on a current Debian bookworm.  I tried to resolve the the first one by do "sudo apt install software-properties-common. No luck still doesn't work.

**Comment by TheAssassin** [comment]
The PPA has never been meant for Debian in the first place. They're specific to Ubuntu and it's one major difference between Ubuntu and Debian.

Just use the continuous release `.deb` files in the meantime. You can `sudo dpkg -i` the `.deb` file. If there's dependency problems, run `sudo apt install -f` afterwards. The latter command pulls in the necessary dependencies from Debian.
